src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,27 @@
+$card-width: 200px;
```

Comment:
Why do you have 2 variables $card-width?

Reply:
Becouse I can't add my scss file with variables to my main scss file, but I've deleted extra variable, now $card-width is only one.